### PR TITLE
condor monitor report fix

### PR DESCRIPTION
**Pull request by ticoann**
@goughes, Erik, I found some minor bug on drain monitoriing. please review.

### CODE REVIEW

File: src/couchapps/WMStats/_attachments/js/minified/import-all-t1.min.js [hunk 1]
```diff
@@ -4402,7 +4402,6 @@ WMStats.namespace('ResubmissionList');
             htmlstr += "<li><b>ACDC DB Name:</b><input type='text' name='ACDCDatabase' size=30  value='wmagent_acdc'/></li>";
         }
         htmlstr += "<li><b>RequestPriority:</b> " + summary.RequestPriority + "</li>";
-        htmlstr += "<li><b>RequestType:</b> " + summary.RequestType + "</li>";
```
Comment (goughes) [inline src/couchapps/WMStats/_attachments/js/minified/import-all-t1.min.js]: This line always gets removed when running combineMinifyWMStats.py, I added it back in the drain stats PR.  Not sure if it is needed or not.

**Comment by goughes** [comment]
Ahh, yes good catch.  I must have forgotten to change this after I added the CondorAPI class.  This patch looks good.  Check my comment about the minify script to see what you think.

File: src/couchapps/WMStats/_attachments/js/minified/import-all-t1.min.js (hunk 1 above)
Comment (ticoann) [reply src/couchapps/WMStats/_attachments/js/minified/import-all-t1.min.js]: Actually, it seems this code was changed manually before. (Not using combinedMinifyWMStats.py) That is why it is still there. This should be removed. Thanks for pointing out.

**Comment by ticoann** [comment]
tested in vm

**Pull request merged**